src/commands/dev/maintenance.ts

```diff
@@ -8,6 +10,10 @@ export const data = new SlashCommandBuilder()
 
 export async function execute(interaction: CommandInteraction) {
     await interaction.deferReply({ ephemeral: true, fetchReply: true })
+    if(!await PermissionUtils.hasPermission(interaction, [PermissionFlagsBits.Administrator], true)) {
+        await interaction.editReply({ embeds: [errorEmbed(interaction, new Error("Vous n'avez pas la permission d'utiliser cette commande."))] })
```

Comment:
src/commands/dev/maintenance.ts#L14

Unsafe call of an `error` type typed value.